src/content/docs/agents/python-agent/getting-started/poetry-break.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Poetry Break'
+contentType: page
+template: basicDoc
+---
+
+##Poem
+
+But a bird that stalks
```

Comment:
@bradleycamacho IIRC you have ordered the stanzas incorrectly. Please reverse. I also think you should include a lead-in sentence.

Reply:
Flipped these!